packages/blocks/shuffle/src/components/item-list.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FunctionComponent, useState } from "react";
+import { List } from "@mui/material";
+import {
+  DndContext,
+  DragOverlay,
+  closestCenter,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+} from "@dnd-kit/core";
+import {
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy,
+} from "@dnd-kit/sortable";
+import { Items } from "../shuffle";
+import { Item } from "./item";
+
+type ItemListProps = {
+  list: Items;
+  onReorder: (sourceIndex: number, destinationIndex: number) => void;
+  onValueChange: (index: number, value: string) => void;
+  onItemBlur: () => void;
+  onAdd: (index: number) => void;
+  onDelete: (index: number) => void;
+};
+
+export const ItemList: FunctionComponent<ItemListProps> = ({
+  list,
+  onReorder,
+  onValueChange,
+  onItemBlur,
+  onAdd,
+  onDelete,
+}) => {
+  const [activeIndex, setActiveIndex] = useState<number | null>(null);
+
+  const sensors = useSensors(
+    useSensor(PointerSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates,
+    }),
```

Comment:
It works for me when tabbing into the drag handle and then pressing Enter. Can you check if that works?
I've since made some improvements to tab navigation between items and the animations so that it's more obvious when sorting with the keyboard.